### PR TITLE
MOSIP-15119 DB Changes for Draft

### PR DESCRIPTION
**Pull request by rambhatt1591**

### CODE REVIEW

**Review by manojsp12 — CHANGES_REQUESTED** [review]

File: db_release_scripts/mosip_idrepo/ddl/idrepo-uin_draft.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	uin_hash character varying (128) NOT NULL,
+	uin_data bytea,		
+	uin_data_hash character varying (64),			
+	status_code character varying (32) NOT NULL,	
+	cr_by character varying (256) NOT NULL,		
+	cr_dtimes timestamp NOT NULL,		
+	upd_by	character varying (256),			
+	upd_dtimes timestamp,			
+	is_deleted bool	DEFAULT FALSE,
+	del_dtimes timestamp FALSE,
```
Comment (manojsp12) [inline db_release_scripts/mosip_idrepo/ddl/idrepo-uin_draft.sql:28]: "FALSE" is not applicable here

File: db_release_scripts/mosip_idrepo/ddl/idrepo-uin_biometric_draft.sql [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	biometric_file_hash character varying(64) NOT NULL,					
+	cr_by character varying(256) NOT NULL,		
+	cr_dtimes timestamp NOT NULL,			
+	upd_by character varying(256),		
+	upd_dtimes timestamp,	
+	is_deleted bool	DEFAULT	FALSE,
+	del_dtimes timestamp,														
+	CONSTRAINT pk_uinbiodft_id PRIMARY KEY (reg_id,biometric_file_type),
+	CONSTRAINT unq_biofileid UNIQUE (bio_file_id),
+	CONSTRAINT unq_uinhsh UNIQUE (uin_hash)
```
[the remark this answers is not in the document]
Comment (rambhatt1591) [reply db_release_scripts/mosip_idrepo/ddl/idrepo-uin_biometric_draft.sql]: Removed.

File: db_release_scripts/mosip_idrepo/ddl/idrepo-uin_document_draft.sql [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+-- 
+-- ------------------------------------------------------------------------------------------
+
+-- object: idrepo.uin_document_draft | type: TABLE --
+-- DROP TABLE IF EXISTS idrepo.uin_document_draft CASCADE;
+CREATE TABLE idrepo.uin_document_draft(
+	reg_id character varying(39) NOT NULL,										
+	doccat_code character varying(36) NOT NULL,									
+	doctyp_code character varying(64) NOT NULL,										
+	doc_id character varying(128) NOT NULL,UNIQUE
```
Comment (manojsp12) [inline db_release_scripts/mosip_idrepo/ddl/idrepo-uin_document_draft.sql:20]: UNIQUE keyword not applicable here

**Review by manojsp12 — CHANGES_REQUESTED** [review]

File: db_release_scripts/mosip_idrepo/ddl/idrepo-uin_draft.sql (hunk 1 above)
Comment (rambhatt1591) [reply db_release_scripts/mosip_idrepo/ddl/idrepo-uin_draft.sql]: Updated.

File: db_scripts/mosip_idrepo/ddl/idrepo-uin_document_draft.sql [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+-- 
+-- ------------------------------------------------------------------------------------------
+
+-- object: idrepo.uin_document_draft | type: TABLE --
+-- DROP TABLE IF EXISTS idrepo.uin_document_draft CASCADE;
+CREATE TABLE idrepo.uin_document_draft(
+	reg_id character varying(39) NOT NULL,										
+	doccat_code character varying(36) NOT NULL,									
+	doctyp_code character varying(64) NOT NULL,										
+	doc_id character varying(128) NOT NULL,UNIQUE
```
Comment (manojsp12) [inline db_scripts/mosip_idrepo/ddl/idrepo-uin_document_draft.sql:20]: UNIQUE not applicable here

File: db_scripts/mosip_idrepo/ddl/idrepo-uin_draft.sql [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+	uin_hash character varying (128) NOT NULL,
+	uin_data bytea,		
+	uin_data_hash character varying (64),			
+	status_code character varying (32) NOT NULL,	
+	cr_by character varying (256) NOT NULL,		
+	cr_dtimes timestamp NOT NULL,		
+	upd_by	character varying (256),			
+	upd_dtimes timestamp,			
+	is_deleted bool	DEFAULT FALSE,
+	del_dtimes timestamp FALSE,
```
Comment (manojsp12) [inline db_scripts/mosip_idrepo/ddl/idrepo-uin_draft.sql:28]: FALSE not applicable here

**Review by manojsp12 — CHANGES_REQUESTED** [review]

**Pull request merged**